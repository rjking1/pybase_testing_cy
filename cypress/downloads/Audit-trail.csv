Date Time,Description
2021-04-23 18:57:43,Heather logged in
2021-04-23 18:57:07,rk logged in
2021-04-23 18:57:04,Heather logged in
2021-04-23 18:55:43,rk logged in
2021-04-23 18:55:40,Heather logged in
2021-04-23 18:55:10,rk logged in
2021-04-23 18:55:03,Heather logged in
2021-04-23 18:53:55,rk logged in
2021-04-16 22:43:20,receipt "75" updated by Heather
2021-04-16 22:42:48,Heather logged in
2021-04-16 22:26:36,emailed rjking58@gmail.com subject test response OK