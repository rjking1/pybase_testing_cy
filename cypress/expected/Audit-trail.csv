Date Time,Description,Key_,Value_
2021-08-10 15:07:13,Janet logged in,,
2021-08-10 15:06:24,Janet logged in,,
2021-08-08 18:38:30,,email.replyTo,Heather King
2021-08-08 18:38:30,,email.subject,Header and footer
2021-08-08 18:38:30,,email.from,Shirley Dougan
2021-08-08 18:35:08,,email.from,Heather King
2021-08-08 18:35:08,,email.replyTo,Shirley Dougan
2021-08-08 18:35:08,,email.subject,Header and footer
2021-08-08 15:21:53,emailed Avanthi <09-avanthi@gmail.com> subject Welcome response,email.from,Heather King
2021-08-08 15:21:48,emailed Pia subject Welcome response,email.replyTo,Shirley Dougan
2021-08-08 15:21:44,emailed Jock subject Welcome response,email.from,Shirley Dougan
2021-08-08 15:21:35,emailed Ilonka subject Welcome response,,
2021-08-08 15:21:30,emailed Jacqueline subject Welcome response,,
2021-08-08 15:21:25,emailed Sharon subject Welcome response,,
2021-08-08 15:16:42,Heather logged in,,
2021-08-08 14:53:14,Heather logged in,,
2021-08-08 14:52:58,Heather logged in,,
2021-08-08 14:51:58,Heather logged in,,
2021-08-08 14:44:39,Heather logged in,,
2021-08-08 14:43:22,member "Jacqueline" updated by Heather,,
2021-08-08 14:40:49,Heather logged in,,
2021-08-08 14:40:30,New Members: Welcome pack sent: "Jacqueline" by Heather,,
2021-08-08 14:39:13,Heather logged in,,
2021-08-08 14:34:33,emailed Kim subject Welcome to FTGAS response,,
2021-08-08 14:34:29,emailed Ram subject Welcome to FTGAS response,,
2021-08-08 14:34:24,emailed Ross subject Welcome to FTGAS response,,
2021-08-08 14:34:19,emailed Barbra subject Welcome to FTGAS response,,
2021-08-08 14:34:15,emailed Betty subject Welcome to FTGAS response,,
2021-08-08 14:34:10,emailed Gerard subject Welcome to FTGAS response,,
2021-08-08 14:34:06,emailed Wendy subject Welcome to FTGAS response,,
2021-08-08 14:34:01,emailed Janice subject Welcome to FTGAS response,,
2021-08-08 14:33:56,emailed Heather subject Welcome to FTGAS response,,
2021-08-08 14:28:48,Heather logged in,,
2021-08-08 14:27:20,Heather logged in,,
2021-08-08 14:18:01,Heather logged in,,
2021-08-06 13:23:28,member "Hermione " added by Heather,,
2021-08-06 13:17:14,Heather logged in,,
2021-08-06 13:15:28,member "Bronwyn" added by Heather,,
2021-08-06 13:08:39,Heather logged in,,
2021-08-06 12:55:26,Heather logged in,,
2021-08-06 12:44:53,Heather logged in,,
2021-08-06 12:38:19,emailed Heather subject Welcome pack response,,
2021-08-06 12:36:06,py_template "Welcome pack" updated by Heather,,
2021-08-06 12:30:39,Heather logged in,,
2021-08-06 12:21:33,Heather logged in,,
2021-08-06 11:11:45,py_template "Welcome pack" updated by Heather,,
2021-08-06 11:07:58,Heather logged in,,
2021-08-05 22:38:28,emailed Shirley subject Welcome Pack Test Email response,,
2021-08-05 22:06:10,emailed Heather subject test welcome pack response,,
2021-08-05 21:41:36,py_template "Welcome pack" updated by Heather,,
2021-08-05 21:38:16,py_template "Welcome pack" updated by Heather,,
2021-08-05 21:29:15,Heather logged in,,
2021-08-05 20:25:03,Heather logged in,,
2021-08-05 00:11:29,py_template "Header and footer" updated by Heather,,
2021-08-05 00:09:26,Templates: Copy selected template(s): "Open Friday Workshops (Still Friendly) and test" by Heather,,
2021-08-05 00:08:31,Heather logged in,,
2021-08-05 00:06:17,py_template "Welcome pack" updated by Heather,,
2021-08-05 00:05:30,Templates: Copy selected template(s): "Foothills art trail" by Heather,,
2021-08-05 00:03:27,Heather logged in,,
2021-08-04 23:44:47,py_template "header and footer" updated by Heather,,
2021-08-04 23:40:20,Heather logged in,,
2021-08-04 23:35:57,Heather logged in,,
2021-08-04 23:29:23,Heather logged in,,
2021-08-04 23:15:35,emailed Liesl subject Open Friday workshop response,,
2021-08-04 23:15:31,emailed Jasmine subject Open Friday workshop response,,
2021-08-04 23:15:28,emailed Roberta subject Open Friday workshop response,,
2021-08-04 23:15:24,emailed Nancy subject Open Friday workshop response,,
2021-08-04 23:15:17,emailed Diane (Ann) subject Open Friday workshop response,,
2021-08-04 23:15:13,emailed Mike subject Open Friday workshop response,,
2021-08-04 23:15:10,emailed Louise subject Open Friday workshop response,,
2021-08-04 23:15:06,emailed Bella subject Open Friday workshop response,,
2021-08-04 23:15:02,emailed Annette subject Open Friday workshop response,,
2021-08-04 23:14:59,emailed Julie subject Open Friday workshop response,,
2021-08-04 23:14:55,emailed Marlene subject Open Friday workshop response,,
2021-08-04 23:14:52,emailed Josephine subject Open Friday workshop response,,
2021-08-04 23:14:48,emailed Leanne subject Open Friday workshop response,,
2021-08-04 23:14:44,emailed Rob subject Open Friday workshop response,,
2021-08-04 23:14:41,emailed Margaret subject Open Friday workshop response,,
2021-08-04 23:14:37,emailed Geoff subject Open Friday workshop response,,
2021-08-04 23:14:33,emailed Ann Maree subject Open Friday workshop response,,
2021-08-04 23:14:29,emailed Robert subject Open Friday workshop response,,
2021-08-04 23:14:26,emailed Robin subject Open Friday workshop response,,
2021-08-04 23:14:22,emailed Kaitlyn subject Open Friday workshop response,,
2021-08-04 23:14:18,emailed Anita subject Open Friday workshop response,,
2021-08-04 23:14:15,emailed Peter subject Open Friday workshop response,,
2021-08-04 23:14:11,emailed Karen subject Open Friday workshop response,,
2021-08-04 23:14:07,emailed David subject Open Friday workshop response,,
2021-08-04 23:14:04,emailed Teresa subject Open Friday workshop response,,
2021-08-04 23:14:00,emailed Elle subject Open Friday workshop response,,
2021-08-04 23:13:56,emailed Georgia subject Open Friday workshop response,,
2021-08-04 23:13:53,emailed Anne subject Open Friday workshop response,,
2021-08-04 23:13:49,emailed Rachel subject Open Friday workshop response,,
2021-08-04 23:13:46,emailed Jacqueline subject Open Friday workshop response,,
2021-08-04 23:13:42,emailed Gillian subject Open Friday workshop response,,
2021-08-04 23:13:38,emailed David subject Open Friday workshop response,,
2021-08-04 23:13:35,emailed Sharon subject Open Friday workshop response,,
2021-08-04 23:13:31,emailed Imogen subject Open Friday workshop response,,
2021-08-04 23:13:27,emailed Robin Leslie subject Open Friday workshop response,,
2021-08-04 23:13:24,emailed Linda subject Open Friday workshop response,,
2021-08-04 23:13:20,emailed Dorothy subject Open Friday workshop response,,